smoke/IotEdgeQuickstart/details/IotedgedWindows.cs

```diff
@@ -135,24 +135,30 @@ public async Task Configure(string connectionString, string image, string hostna
 
         public async Task Start()
         {
-            // In Windows 10 RS4, after service is configured; it doesn't start up automatically
-            // Therefore we need to explicitly start it.
-            using (var cts = new CancellationTokenSource(TimeSpan.FromMinutes(2)))
+            Console.WriteLine($"Current OS Version: {Environment.OSVersion}");
+
+            // Need to start up service explicitly as service is not started up automatically after configured in Windows 10 RS4
+            if (Environment.OSVersion.ToString().Equals("Microsoft Windows NT 6.2.9200.0"))
             {
-                try
+                Console.WriteLine($"Starting up iotedged on Windows");
+
+                using (var cts = new CancellationTokenSource(TimeSpan.FromMinutes(2)))
                 {
-                    await Process.RunAsync("sc", "start iotedge", cts.Token);
+                    try
+                    {
+                        await Process.RunAsync("sc", "start iotedge", cts.Token);
```

Comment:
Since you're already using `ServiceController`, you should use its `.Start()` function rather than shelling out to `sc.exe`